demisto_sdk/commands/content_graph/common.py

```diff
@@ -276,6 +276,81 @@ def by_schema(path: Path, git_sha: Optional[str] = None) -> "ContentType":
                     return content_type
         raise ValueError(f"Could not find content type in path {path}")
 
+    @property
+    def convert_content_type_to_rn_header(self) -> str:
+        """
+        Convert ContentType to the Release note header.
+        """
+        if self == ContentType.PREPROCESS_RULE:
+            return "PreProcess Rules"
+        elif self == ContentType.TRIGGER:
+            return "Triggers Recommendations"  # https://github.com/demisto/etc/issues/48153#issuecomment-1111988526
+        elif self == ContentType.XSIAM_REPORT:
```

Comment:
we can do pascalToSpace to many of the types here (need to add this method to tools.py)